provider/azure/internal/imageutils/images.go

```diff
@@ -94,7 +94,7 @@ func SeriesImage(
 		publisher = centOSPublisher
 		offering = centOSOffering
 		switch series {
-		case "centos7":
+		case "centos7", "centos8":
 			sku = "7.3"
```

Comment:
Is this sku right for centos8?

Reply:
Appears so, i tested it live.
I couldn't find any definite docs to what what the SKU should be for centos7 vs 8